waterlog/src/actions/SegmentEvents.js

```diff
@@ -1,12 +1,15 @@
 import { FETCH_SEGMENTS_EVENTS } from "./types";
 
 export const fetchSegmentsEvents = () => dispatch => {
-    fetch('https://localhost:44382/api/segmentevents')//Change to use either localhost/server
-        .then(res => res.json())
-        .then(events =>
-            dispatch({
-                type: FETCH_SEGMENTS_EVENTS,
-                payload: events
-            })
-        );
-};
\ No newline at end of file
+  fetch("https://localhost:44382/api/segmentevents") //Change to use either localhost/server
+    .then(res => res.json())
+    .then(events =>
+      dispatch({
```

Comment:
do you have to return this dispatch, just saw you are returning the dispatch on the other files

Reply:
I removed the return.

Reply:
File deleted